### DISCUSSION TITLE
make larger/smaller by keeping key presed

### DISCUSSION THREAD

**Discussion by Ana06** [rxhanson/Rectangle#469] [Ideas]
I have just installed Rectangle. It is amazing! ❤️

I wonder if it would be possible to continue increasing/decreasing the window size by keeping the shortcut for `Make Larger/Smaller` pressed, as a fast way to increase/decrease a windows size several times. I would love this feature.

**Comment by rxhanson** [comment]
Thanks for the kind feedback!

Right now, the keyboard shortcut recorder that Rectangle uses (MASShortcut) doesn't support listening to both down and up events for keyboard shortcut execution. I've been considering either modifying MASShortcut or migrating to the shortcut recorder from Sindre Sorhus, which would allow this, but I don't have a timeframe for it. If anyone wants to try it out, I would definitely be inclined to merge a PR.
